packages/base/package.json

```diff
@@ -25,9 +25,8 @@
   "scripts": {
     "build": "npm run build:types && npm run build:js",
     "build:types": "tsc --declaration --declarationMap --emitDeclarationOnly",
-    "build:js": "npm run build:old && babel --root-mode upward src/blockchain.ts --out-file lib/blockchain.js --source-maps",
-    "build:old": "shx mkdir -p lib && shx cp -r src/*.d.ts lib && shx cp -r src/*.js lib",
-    "fmt": "prettier --write \"{lib,tests}/**/*.js\" index.d.ts package.json",
+    "build:js": "babel --root-mode upward src --out-dir lib --source-maps --extensions \".ts\" --ignore \"**/*.d.ts\"",
+    "fmt": "prettier --write \"{lib,tests}/**/*.js\" package.json",
     "lint": "eslint -c ../../.eslintrc.js \"{lib,tests}/**/*.js\"",
     "test:ts": "ava **/*.test.ts --timeout=2m",
```

Comment:
because of the base pkg is refactored in TS, the scripts can be same with other pure ts pkg

```json
{
    "fmt": "prettier --write \"{src,tests,examples}/**/*.ts\" package.json",
    "lint": "eslint -c ../../.eslintrc.js \"{src,tests,examples}/**/*.ts\"",
    "test": "ava **/*.test.ts --timeout=2m",
    "build": "npm run build:types && npm run build:js",
    "build:types": "tsc --declaration --emitDeclarationOnly",
    "build:js": "babel --root-mode upward src --out-dir lib --extensions .ts -s",
    "clean": "rm -rf lib",
    "prepublishOnly": "yarn run clean && yarn run build",
}
```